FeedStoreChallenge/ManagedCache.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Beniamin Idziak on 15/08/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedCache)
+class ManagedCache: NSManagedObject {
```

Comment:
This class is not intended to be subclassed, so it can be final:

```suggestion
final class ManagedCache: NSManagedObject {
```